x-pack/plugins/cases/server/client/sub_cases/update.ts

```diff
@@ -0,0 +1,407 @@
+/*
```

Comment:
Moved this to its own file because it's larger. Most of the code is the same but I had to change some parameter names.